src/lib/journey/callbacks/password/password.stories.js

```diff
@@ -48,3 +58,22 @@ Interaction.play = async ({ canvasElement }) => {
   await userEvent.click(eye);
   expect(cb.getInputValue()).toBe('password');
 };
+
+export const InteractionWithCheckbox = Template.bind({});
+
+InteractionWithCheckbox.args = { ...PasswordWithCheckbox.args };
+
+InteractionWithCheckbox.play = async ({ canvasElement }) => {
```

Comment:
This is great. Thanks! I'd like to add two more tests: 

1. One that ensures `'button'` works
2. One that ensure the new disable option works with `'none'`

Reply:
the one that ensures button works is the ona called 'base'. It sets the style to 'button'
```
export const Base = {
  args: {
    callback: step.getCallbackOfType(CallbackType.PasswordCallback),
    **style: {
      showPassword: 'button',
    },**
  },
};
```
I will add another one with the option 'none'